.github/workflows/test-coverage.yml

```diff
@@ -144,3 +138,4 @@ jobs:
           flags: unittests
           name: codecov-umbrella
           verbose: true
+          
```

Comment:
There is a missing newline here. Or maybe there's a newline but it's followed by spaces: please remove the spaces then.

The GitHub view of this PR must not say "no newline at the end of file"